packages/embed/src/components/pausedpopover/ListenOnAudiusCTA.jsx

```diff
@@ -1,26 +1,34 @@
+import { FixedDecimal } from '@audius/fixed-decimal'
+import { Button, Text } from '@audius/harmony'
 
-import AudiusLogo from '../../assets/img/audiusLogoHorizontal.svg'
 import { getCopyableLink } from '../../util/shareUtil'
-import Button from '../button/Button'
-
-import styles from './ListenOnAudiusCTA.module.css'
 
 const messages = {
-  label: 'Listen on'
+  listen: 'Listen on Audius',
+  buy: 'Buy'
 }
 
-const ListenOnAudiusCTA = ({ audiusURL }) => {
+const ListenOnAudiusCTA = ({ audiusURL, premiumConditions }) => {
   const onClick = () => {
     window.open(getCopyableLink(audiusURL), '_blank')
   }
+  const isPurchaseable =
+    premiumConditions && 'usdc_purchase' in premiumConditions
 
   return (
     <Button
+      color={isPurchaseable ? 'lightGreen' : undefined}
+      fullWidth
       onClick={onClick}
-      className={styles.container}
-      icon={<AudiusLogo />}
-      label={messages.label}
-    />
+    >
+      <Text variant='title' size='l'>
+        {isPurchaseable
+          ? `${messages.buy} $${new FixedDecimal(
+              premiumConditions.usdc_purchase.price / 100
+            ).toLocaleString('en-US', { minimumFractionDigits: 2 })}`
```

Comment:
<3 Though this is pretty safe since we're using cents and dividing by 100, if you want extra max safety add `maximumFractionDigits: 2`

Reply:
MAXIMUM! yes

Reply:
resolved offline, using `USDC` instead of raw `FixedDecimal`